lib/taskana-core-test/src/test/java/acceptance/classification/get/GetClassificationAccTest.java

```diff
@@ -0,0 +1,155 @@
+    Classification classification =
+        classificationService.getClassification(classificationWithAllProperties.getId());
+
+    assertThat(classification)
+        .hasNoNullFieldsOrProperties()
+        .isEqualTo(classificationWithAllProperties);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_ThrowException_When_ClassificationIdIsNull() {
+    assertThatThrownBy(() -> classificationService.getClassification(null))
+        .isInstanceOf(ClassificationNotFoundException.class);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_ThrowException_When_ClassificationIdDoesNotExist() {
+    assertThatThrownBy(() -> classificationService.getClassification("CLI:NonExistingId"))
+        .isInstanceOf(ClassificationNotFoundException.class);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_getClassificationByKeyAndDomain_When_AllPropertiesSet() throws Exception {
+    Classification classification =
+        classificationService.getClassification(
+            classificationWithAllProperties.getKey(), classificationWithAllProperties.getDomain());
+
+    assertThat(classification).isEqualTo(classificationWithAllProperties);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_ThrowException_When_ClassificationKeyIsNull() {
+    assertThatThrownBy(
+            () ->
+                classificationService.getClassification(
+                    null, classificationWithAllProperties.getDomain()))
+        .isInstanceOf(ClassificationNotFoundException.class);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_ThrowException_When_ClassificationKeyDoesNotExist() {
+    assertThatThrownBy(
+            () ->
+                classificationService.getClassification(
+                    "WrongKey", classificationWithAllProperties.getDomain()))
+        .isInstanceOf(ClassificationNotFoundException.class);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_ReturnFromMasterDomain_When_ClassificationDomainDoesNotExist() throws Exception {
+    Classification classification =
+        classificationService.getClassification(
+            classificationWithAllProperties.getKey(), "WrongDomain");
+
+    assertThat(classification).isNotNull().hasNoNullFieldsOrProperties();
+    assertThat(classification.getDomain()).isEmpty();
+    assertThat(classification.getKey()).isEqualTo(classification.getKey());
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_GetClassification_When_NameContainsSpecialCharacter() throws Exception {
+    Classification classification =
+        DefaultTestEntities.defaultTestClassification()
+            .name("Zustimmungserklärung")
+            .buildAndStore(classificationService, "businessadmin");
+
+    Classification result = classificationService.getClassification(classification.getId());
+
+    assertThat(result.getName()).isEqualTo("Zustimmungserklärung");
+  }
```

Comment:
what are we exactly testing here? That the unicode character gets set properly?
if so make that clear by its method name?
exaple:
should_GetSpecialCharacterCorretly_When_AFieldContainsASpecialCharacter

Reply:
I will rename the method